tests/test_smoke.py

```diff
@@ -325,23 +331,50 @@ def test_env_check():
 # ---------- file_mounts ----------
 def test_file_mounts():
     name = _get_cluster_name()
+    test_commands = [
+        *storage_setup_commands,
+        f'sky launch -y -c {name} examples/using_file_mounts.yaml',
+        f'sky logs {name} 1 --status',  # Ensure the job succeeded.
+    ]
     test = Test(
         'using_file_mounts',
-        [
-            'touch ~/tmpfile',
-            'mkdir -p ~/tmp-workdir',
-            'touch ~/tmp-workdir/tmp\ file',
-            'touch ~/tmp-workdir/foo',
-            'ln -f -s ~/tmp-workdir/ ~/tmp-workdir/circle-link',
-            f'sky launch -y -c {name} examples/using_file_mounts.yaml',
-            f'sky logs {name} 1 --status',  # Ensure the job succeeded.
-        ],
+        test_commands,
         f'sky down -y {name}',
         timeout=20 * 60,  # 20 mins
     )
     run_one_test(test)
 
 
+# ---------- storage ----------
+def test_storage_mounts():
+    name = _get_cluster_name()
+    yaml_str = pathlib.Path(
+        'tests/test_yamls/test_storage_mounting.yaml').read_text()
+    storage_name = f'sky-test-{int(time.time())}'
+    yaml_str = yaml_str.replace('random_bucket_name', storage_name)
```

Comment:
The design choice to replace `random_bucket_name` with another name stands out a bit. Running the yaml file standalone will def not work as `random_bucket_name` is most likely taken, so it probably better to make it a jinja file. Why can't we make `test_storage_mounting.yaml` a jinja file instead and fill in the placeholder variable?

Reply:
Done for `test_storage_mounting`! I didn't do jinja for `test_spot_storage` because it uses a YAML in the examples folder, and it might be confusing for users to see jinja templates in the examples.